storage/innobase/xtrabackup/src/ddl_tracker.cc

```diff
@@ -548,6 +556,12 @@ dberr_t ddl_tracker_t::handle_ddl_operations() {
     }
   }
 
+  for (auto &table : missing_after_discovery) {
```

Comment:
@aybek Do you know why need this? We will not add any table which is missing to tables_copied_no_lock. isn't it? :thinking:

Reply:
Fixed by the new commit